src/main/java/com/iridium/iridiumskyblock/configs/Configuration.java

```diff
@@ -31,6 +31,7 @@
     public String dateTimeFormat = "EEEE, MMMM dd HH:mm:ss";
     public String spawnWorldName = "world";
     public String defaultCommandSyntax = "/is %command%";
+    public String islandLevelEquation = "abs(cbrt($1+1))";
```

Comment:
Why is there an abs() call? All cube roots are positive

Reply:
:shrug: It was in my first level equation code and I figured there was probabbly a reason, and if it is useless, it does litrally nothing so meh